packaging/ubuntu-16.04/snapd.install

```diff
@@ -17,7 +17,7 @@ data/udev/rules.d/66-snapd-autoimport.rules /lib/udev/rules.d
 data/info /usr/lib/snapd/
 
 # snap-confine stuff
-etc/apparmor.d/usr.lib.snapd.snap-confine.real
+# etc/apparmor.d/usr.lib.snapd.snap-confine.real
```

Comment:
Perhaps we should remove the top-level symlink and just add `packaging/debian-9` and let the CI part add the relevant stuff. Note that for fedora the process is totally different so we need to re-think how this is done.